sanic_routing/group.py

```diff
@@ -68,55 +133,26 @@ def merge(self, group, overwrite: bool = False, append: bool = False):
         self._routes = tuple(_routes)
 
     @property
-    def labels(self):
-        return self[0].labels
+    def depth(self) -> int:
+        return len(self[0].parts)
 
     @property
-    def methods(self):
-        return frozenset(
-            [method for route in self for method in route.methods]
+    def dynamic_path(self) -> bool:
+        return any(
+            (param.label == "path") or (r"/" in param.label)
```

Comment:
Unnecessary raw string for forward slash

Reply:
Fixed in next push.